ethers-solc/src/remappings.rs

```diff
@@ -276,8 +320,22 @@ impl RelativeRemapping {
 // Remappings are printed as `prefix=target`
 impl fmt::Display for RelativeRemapping {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let mut s = {
-            #[cfg(windows)]
+        let mut s = String::new();
+        if let Some(context) = self.context.as_ref() {
+            #[cfg(target_os = "windows")]
```

Comment:
we have tests for windows style paths but did you get to test this on a windows box? don't have one available atm :/

Reply:
Tested in CI